application/onboarding/controller.py

```diff
@@ -532,9 +536,12 @@ def verify_account_random_deposit():
                 'class': 'info',
                 'content':'Bank account has been verified'})
             return redirect(url_for('.application_complete'))
+        except error.UserInputError as e:
+            flash(e.message)
```

Comment:
That was intentional, the stripe call error is being logged in the stripe_client.py, here for UserInputError it is informational